Makefile

```diff
@@ -168,8 +168,10 @@ undeploy:
 #
 # You must also set:
 # ELASTICSEARCH_OPERATOR_NAMESPACE (Default: openshift-operators-redhat)
-test-e2e-olm: gen-example-certs
-	hack/test-e2e-olm.sh
+TEST_NAMESPACE="$${TEST_NAMESPACE:-e2e-test-$${RANDOM}}"
+test-e2e-olm: DEPLOYMENT_NAMESPACE="${TEST_NAMESPACE}"
+test-e2e-olm: gen-example-certs 
+	TEST_NAMESPACE=${TEST_NAMESPACE} hack/test-e2e-olm.sh
 
```

Comment:
The result will not be as expected with this syntax. First of all, the env var RANDOM will be read multiple times resulting in additional unique namespaces on each each read invocation, whether we want a single TEST_NAMESPACE. Furthermore, providing a Makefile override to TEST_NAMESPACE is simpler with `?=`. I propose to changes these lines to:
```
RANDOM_SUFFIX:=$(shell echo $$RANDOM)
TEST_NAMESPACE?="e2e-test-$(RANDOM_SUFFIX)"
test-e2e-olm: DEPLOYMENT_NAMESPACE=$(TEST_NAMESPACE)
test-e2e-olm: gen-example-certs
	TEST_NAMESPACE=$(TEST_NAMESPACE) hack/test-e2e-olm.sh
```